src/main/resources/templates/subscription.html

```diff
@@ -156,7 +156,22 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     </label>
                                 </div>
                             </div>
-
+                            
+                             <!-- Form post emailSubject field -->                            
+                            <div class="form-group" data-bind="visible: $data.notificationType() == 'MAIL'">
+                                <label class="pl-1 control-label font-weight-bold">Subject</label>
+                                <img class="cursor-pointer popup-large" id="restPostMediaTypeInfo" alt="RestPostMediaType Info" src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
+                                    title=
+                                        "This is your choice for the email notification subject. <br>
+                                        Default value is 'Email Subscription Notification' <br>
+                                        which is configured in the property file" />                                
+                                <div >
+                                    <label>
+                                        <input name="emailSubject"  id="emailSubjectID" placeholder="emailSubject" type="text"/>                                        
```

Comment:
We should discuss the width of the input in the team. All other inputs stretch 100% ish, Maybe this should be set here to, or Others may be changed.

Reply:
done